chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -83,6 +83,9 @@
   # Stop remaining pacemaker resources
   bash "stop pacemaker resources" do
     code <<-EOF
+      crm stop postgresql
+      # Delete postgres data directory
```

Comment:
The postgres data dir needs to be deleted after we stop postgresql server but before we stop fs-postgresql service.